cmd/ww/main.go

```diff
@@ -87,8 +88,12 @@ func setup(c *cli.Context) (err error) {
 	} else if env.Host, err = libp2p.New(); err != nil {
 		return
 	}
-	if env.DHT, err = dual.New(c.Context, env.Host); err != nil {
 
+	env.DHT, err = dual.New(c.Context, env.Host,
+		dual.WanDHTOption(dht.BootstrapPeersFunc(env.PublicBootstrapPeers)),
+		dual.LanDHTOption(dht.BootstrapPeersFunc(env.PrivateBootstrapPeers)))
```

Comment:
Function is ultimately called in a loop [here](https://github.com/libp2p/go-libp2p-kad-dht/blob/master/dht.go#L474-L495), by `fixLowPeers()`, at 2min intervals.